lib/mapper.js

```diff
@@ -189,14 +205,18 @@ function properties(track){
 
 function traits(identify){
```

Comment:
Sweet, just checked again and we're doing the same thing over at MC. 

Letting users override via code if theyneed to and falling back on the settings!

@sperand-io if looks good, can you merge? I'll deploy